src/Razor/src/Microsoft.VisualStudioCode.RazorExtension/Services/LspDynamicFileProvider.LspTextChangesTextLoader.cs

```diff
@@ -45,11 +45,22 @@ public override async Task<TextAndVersion> LoadTextAndVersionAsync(LoadTextOptio
             var sourceText = await _document.GetTextAsync(cancellationToken).ConfigureAwait(false);
 
             // Validate the checksum information so the edits are known to be correct
-            if (IsSourceTextMatching(sourceText))
+            try
```

Comment:
It seems possible that the failure is on line 41 as well?  I couldn't see line numbers in the stack trace.

Reply:
It _shouldnt_ be possible but here we are so may be worth wrapping it all.